src/app/demo/inline/inline.component.html

```diff
@@ -0,0 +1,290 @@
+<div>
```

Comment:
I don't think it is a good idea to have another demo page for the inline component.  There is too much duplicated code here. Let's add it to the current demo page.